server/package.json

```diff
@@ -4,6 +4,7 @@
     "main": "index.js",
     "scripts": {
         "start": "ts-node-dev --respawn src/index.ts",
+        "start:local": "DB_ENV=local ts-node-dev --respawn src/index.ts",
```

Comment:
I believe this only works on Linux and Mac - see https://stackoverflow.com/questions/40806796/setting-environment-variables-package-json-in-windows-10, I was getting the same error.

Adding another script for windows `start:local: set DB_ENV=local && ts-node-dev --respawn src/index.ts` should fix it.

Reply:
Additionally, we can do npm run start:local - no need to use yarn commands

Reply:
I don't think we can have two scripts with the same name. Also, it works for me even though I'm on windows but I'm using bash which is probably why. I think most devs on windows use bash or some unix terminal anyway?

Reply:
name it something with windows i guess. I have bash too and it doesnt work.

Reply:
Windows is so godamnn frustrating.

Reply:
That's interesting b/c the script you provided with the `set` command doesn't work for me haha apparently the variable never gets set for some reason.

Reply:
THATS MY PROBLEM WTF? LOL

Reply:
This reminds me of the docker issues...

Reply:
yea this is rly strange lol we must have something configured differently haha